src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js

```diff
@@ -0,0 +1,13 @@
+(function () {
+    const editButton = document.querySelector('.ez-btn--edit');
+    const languageRadioOption = document.querySelector('.ez-extra-actions--edit.ez-extra-actions--prevent-show [type="radio"]');
+
+    if (null === languageRadioOption) {
+        return;
+    }
+
+    editButton.addEventListener('click', () => {
+        languageRadioOption.checked = true;
+        languageRadioOption.dispatchEvent(new CustomEvent('change'))
+    }, false);
+})();
```

Comment:
To be consistent with other js, we pass `window` and `document` to the method and then use `global` and `doc` inside